src/types/types.ts

```diff
@@ -2591,6 +2591,93 @@ export interface DebugPrestate {
   storage: Record<string, string>;
 }
 
+/*
+ * SDK Representation of a user operation and bundler types
+ */
+export interface UserOperation {
+  sender: string;
+  nonce: string;
+  initCode: string;
+  callData: string;
+  callGasLimit: number;
+  verificationGasLimit: number;
+  preVerificationGas: number;
+  maxFeePerGas: number;
+  maxPriorityFeePerGas: number;
+  paymasterAndData: string;
+  signature: string;
+  entryPoint: string;
+  blockNumber: number;
+  blockHash: string;
+  transactionHash: string;
+}
+
+export interface UserOperationRequest {
+  sender: string;
+  nonce: string;
+  initCode: string;
+  callData: string;
+  callGasLimit: number;
+  verificationGasLimit: number;
```

Comment:
do any of the `number` fields here overflow? if so, we should represent them as BigNumber for now, and i'll replace it with BigInt for v6.